libpy/durable/engine.py

```diff
@@ -456,11 +457,11 @@ def action_callback(e):
                                     
                     except BaseException as error:
                         t, v, tb = sys.exc_info()
-                        print('base exception type {0}, value {1}, traceback {2}'.format(t, str(v), traceback.format_tb(tb)))
+                        logging.error('base exception type {0}, value {1}, traceback {2}'.format(t, str(v), traceback.format_tb(tb)))
```

Comment:
When there is an exception, you have `logging.exception` see https://docs.python.org/3/library/logging.html?highlight=logging%20exception#logging.exception

Reply:
Switched error logs to `logging.exception`